### ISSUE TITLE
[Idea] `tea --request`

### ISSUE THREAD

**Issue by Olshansk** [teaxyz/cli#455]
_This is just an idea/suggestion. If it gets enough 👍, we can think of a design and implementation plan._

## tl;dr

Allow requesting a package from the CLI.

## The Issue

I've found myself in this situation a couple of times today:
<img width="304" alt="Screenshot 2023-03-19 at 2 48 19 PM" src="https://user-images.githubusercontent.com/1892194/226211722-8a4b4b74-90ef-4428-b346-158497737d93.png">

### Questions

I understand that this is a WIP and all the packages will be added over time, but I don't know:
1. Where is the list of backlog packages?
2. How to upvote / promote existing backlog items?
3. How to request new packages to be opened?

## Scope of work

This isn't difficult, but would also require server maintenance, so it can be designed and scoped out assuming it is a highly requested feature.

**Comment by getneil** [comment]
then it would also help in recruiting people to package it.

**Comment by mxcl** [comment]
> Where is the list of backlog packages?
> How to upvote / promote existing backlog items?
> How to request new packages to be opened?

https://github.com/teaxyz/pantry

Our docs refer to this in several places, probs that can be improved but we do document it.

---

I'm not really sure what `tea --request` is for…? But perhaps you are looking for `tea +tea.xyz/brewkit pkg search foo`?

**Comment by Olshansk** [comment]
@mxcl Makes sense to me and I trust your gut sense on what's necessary for a good package manager.

What I'm looking for is a low barrier to entry to requesting packages I use often but don't want to spend my time opening a PR each time. For example, I'm going to use brew for this right now:

![Screenshot 2023-03-24 at 1 49 48 PM](https://user-images.githubusercontent.com/1892194/227637451-6593a71a-cbbc-462f-933e-7ff665a81f99.png)

**Comment by jhheider** [comment]
Doesn't address your issue, but `tea -S k9s` should get the [newly-completed](https://github.com/teaxyz/pantry/pull/853) `k9s` package. ;)

**Comment by davdroman** [comment]
As a side note, might be best to move this to Discussions -> Ideas.

**Comment by Olshansk** [comment]
Moving this out to a discussion in #475 and close this out.
